src/Microsoft.SqlTools.ServiceLayer/LanguageServices/LanguageService.cs

```diff
@@ -1738,6 +1778,8 @@ internal async Task<ScriptFileMarker[]> GetSemanticMarkers(ScriptFile scriptFile
                 out connInfo);
             var parseResult = await ParseAndBind(scriptFile, connInfo);
 
+            await CheckForNonTSqlLanguage(scriptFile.ClientUri, parseResult);
```

Comment:
given that nothing depends on the return value, we probably don't need to `await` this.